lib/reporter.js

```diff
@@ -93,6 +93,7 @@ module.exports = async (data, templateFile, outputFile, showUnique) => {
         const report = await generateTemplate(modifiedData, templateFile);
 
         await writeReport(report, outputFile);
+        return Promise.resolve(modifiedData);
     } catch (err) {
```

Comment:
Seems like in this case catching error does not make any sense. as of it is handled in `reporter`'s catch. It even make sense to not cover it so that catch will work.